lib/search.js

```diff
@@ -38,6 +38,14 @@ function processAndRecur (html, opts, savedApps, clp) {
     .then((apps) => checkFinished(opts, apps, nextToken, clp));
 }
 
+function encodeQueryData (data) {
```

Comment:
use [querystring.stringify](https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options) for this